### DISCUSSION TITLE
eGPU, airdrop

### DISCUSSION THREAD

**Discussion by ivan-ef** [osy86/HaC-Mini#1029] [Q&A]
I have a few questions about nuc hardware:

Does airdrop only works with apple wifi modules?

Is it possible to use thunderbolt eGPU on both MacOS and Windows without applying thunderbolt patch every time?

**Comment by Medizinmann-MD** [comment]
You need a MacOS compatible WiFi module - not necessarily an Apple module – look here:
[Replacing Wifi Card - HaC Mini Hackintosh (gitbook.io)](https://osy.gitbook.io/hac-mini-guide/post-installation/wifi)
IMHO best bet is BCM94360NG - price- and performance-wise.
[Is BCM94360NG viable? · Issue #197 · osy/HaC-Mini (github.com)](https://github.com/osy/HaC-Mini/issues/197)

Flashing TB-firmware every time isn’t advisable. I would leave it alone if you intend to use the NUC with Windows and MacOS!
AFAIK TB3 works if you just plug in the device before booting!
And I would say there is actually no need for hot-plugging with an eGPU…

**Reply by ivan-ef** [reply]
> You need a MacOS compatible WiFi module - not necessarily an Apple module – look here: [Replacing Wifi Card - HaC Mini Hackintosh (gitbook.io)](https://osy.gitbook.io/hac-mini-guide/post-installation/wifi) IMHO best bet is BCM94360NG - price- and performance-wise. [Is BCM94360NG viable? · Issue #197 · osy/HaC-Mini (github.com)](https://github.com/osy/HaC-Mini/issues/197)


I bought ax200 module. Main goal was to use Wi-Fi 6 in windows. Both Wi-Fi and Bluetooth works in macOS, but airdrop is not.

**Reply by ivan-ef** [reply]
> Flashing TB-firmware every time isn’t advisable. I would leave it alone if you intend to use the NUC with Windows and MacOS! AFAIK TB3 works if you just plug in the device before booting! And I would say there is actually no need for hot-plugging with an eGPU…

All TB deices works in MacOS when I use native TB patch, but TB devices doesn't appear in Windows.
TB works in Windows if I use Legacy mode, TB DOCK works in MacOS, but  EGPU shows driver not installed 
<img width="1042" alt="Снимок экрана 2023-01-30 в 01 56 41" src="https://user-images.githubusercontent.com/67600440/215360580-1ab97df8-2157-4faf-a27d-38105c1a8dcc.png">

**Comment by Medizinmann-MD** [comment]
As I wrote - if you want airdrop to work you need a compatible BT/WiFi module and it is not going to work with an AX200 - never! 
You now either could try to get a compatible BT-USB stick for MacOS or the IMHO very much better and much more hassle free approach you replace the AX200 with i.e. BCM94360NG and use a WiFi 6 (or even 6E) USB dongle with Windows (i.e. Netgear AXE3000) or use a WiFi 6 access point (i.e. Linksys MR7350 - I bought one on ebay a few weeks ago for 55€) and plug it in LAN1.

**Comment by S1999U2** [comment]
A bit late. But I bought a BCM94360NG card on eBay. 
It uses a Broadcom chip that is natively supported in macOS. Airdrop, Apple Watch unlock, continuity etc. work flawlessly. 

Wifi and Bluetooth also work in Windows without a problem (although I don’t remember if I had to install an additional driver for that).

**Reply by S1999U2** [reply]
Also, you don’t have to do any modifications on your NUC with his card.

**Reply by ivan-ef** [reply]
I purchased every possible Broadcom modules include bcm94360cs2 bcm943602cs
Dw1830
I will test all of them with adapters and write a thread with results.

**Reply by ivan-ef** [reply]
> A bit late. But I bought a BCM94360NG card on eBay. It uses a Broadcom chip that is natively supported in macOS. Airdrop, Apple Watch unlock, continuity etc. work flawlessly.

Can't get BCM94360 work in Monterey. Bluetooth chipset is working but wifi is not. 
Removed old BRCM kexts, but wifi adapter is not listed in system report. Could you check if 80211family kext is loading in your system? ('kextstat' command in terminal)
